src/preprocess/acdc_data_split.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import csv
+import logging
+import random
+from collections.abc import Iterable
+from pathlib import Path
+
+
+def main(args):
+    # Randomly and evenly split the data into k folds based on the diseases.
+    data_paths = sorted((args.data_dir / 'training').glob('**/Info.cfg'))
+    group_dict = {
+        'NOR': [],  # healthy patients
+        'MINF': [],  # patients with previous myocardial infarction
+        'DCM': [],  # patients with dilated cardiomyopathy
+        'HCM': [],  # patients with an hypertrophic cardiomyopathy
+        'RV': []  # patients with abnormal right ventricle
+    }
+    for data_path in data_paths:
+        with open(data_path, 'r') as f:
+            for line in f:
+                key, value = line.rstrip('\n').split(': ')
+                if key == 'Group':
+                    group_dict[value].append(data_path.parent.resolve())
+                    break
+    _, groups = zip(*sorted(group_dict.items()))
+    random.seed(0)
+    groups = tuple(random.sample(group, k=len(group)) for group in groups)
+    folds = tuple(zip(*groups))  # twenty folds where a fold contains five types of patients.
```

Comment:
The comment may be a little ambiguous.

There are twenty folds because there are twenty cases in each group. Hence, we can state clearly for those readers who are unfamiliar with the ACDC dataset details.

Reply:
Actually I just want to emphasize that a fold contains five types of patients (means we split evenly), not "there are twenty folds". I can add why there are twenty folds though.

Reply:
OK!

Reply:
Solved in 258d296.